functions/__fzf_grep_current_dir.fish

```diff
@@ -0,0 +1,36 @@
+function __fzf_grep_current_dir --description "Search current directory content for $token. Replace the current token with the selected file paths."
+    # Make sure that fzf uses fish to execute __fzf_preview_file.
+    # See similar comment in __fzf_search_shell_variables.fish.
+    set --local --export SHELL (command --search fish)
+
+    set token (commandline --current-token | string unescape)
+
+    set fzf_arguments --multi --ansi --exact --phony --reverse --query=$token \
+        --preview='rg --ignore-case --pretty --context 10 {q} {}' \
+        --bind "change:reload:rg --ignore-case --files-with-matches --no-ignore-vcs {q} || true"
+
+    if test "$token" = ""
+        set file_paths_selected (rg --files | fzf $fzf_arguments)
+    else
+        set file_paths_selected (rg --files-with-matches $token | fzf $fzf_arguments)
+    end
```

Comment:
This might not be possible, but I'd really like us to not have to make a special case search for starting the search using the token. If only there was a way to automatically execute `reload` as soon as fzf starts up 🤔

Reply:
Btw `rg --files-with-matches ""` gives the same output as `rg --files`, so you don't have to do two cases based on the token.

Reply:
Nice find, that would simplify things a bit. I'll push an update tomorrow